resource_processor/vmss_porter/runner.py

```diff
@@ -137,6 +151,8 @@ async def build_porter_command(msg_body, env_vars):
     installation_id = get_installation_id(msg_body)
 
     command_line = [f"{azure_login_command(env_vars)} && {azure_acr_login_command(env_vars)} && porter "
+                    # If a custom action (i.e. not install, uninstall, upgrade) we need to use 'invoke'
+                    f"{'invoke --action ' if custom_action else ''}"
```

Comment:
I'm not a fan of over-using the suffix if/else, and I think this one is hard to read
Is this adding `invoke --action` for customer actions, and otherwise doing nothing?

Reply:
I think this is the right place to use it as there isn't much logic to evaluate; yes, we're adding 'invoke --action' to the command if its a custom action else leaving it blank. What would you prefer? Or I can adjust the comment to make it clearer?

Reply:
I think I'd have created the token that I was going to insert before I got here (i.e. before line 153)...

``` python

if custom_action:
    invoke_token = "-invoke --action"
else:
    invoke_token = ""
```

and then used this in the command_line buildup...

``` python
f"{invoke_token}"
```

... primarily because I don't like the suffixed if, and I don't like too branching within an f-string.

BUT...this is a style choice on my part, so let's move on :)